extensions/sql-database-projects/src/common/utils.ts

```diff
@@ -70,6 +70,18 @@ export function trimChars(input: string, chars: string): string {
 	return output;
 }
 
+/**
+ * Ensures that folder path terminates with the slash.
+ * Windows-style slash (`\`) is used by default.
+ *
+ * @param path Folder path to ensure trailing slash for.
+ * @param slashCharacter Slash character to ensure is present at the end of the path.
+ * @returns Path that ends with the given slash character.
+ */
+export function ensureTrailingSlash(path: string, slashCharacter: string = '\\'): string {
```

Comment:
This technically allows any character, not just slash, so name is not 100% accurate, but I think it's fine. We can rename it later, if people decide to abuse it for other purposes.